dist/js/jquery.justifiedGallery.js

```diff
@@ -208,7 +208,9 @@
   
       // Image reloading for an high quality of thumbnails
       var imageSrc = $image.attr('src');
-      var newImageSrc = this.newSrc(imageSrc, imgWidth, imgHeight, $image[0]);
+      if (imageSrc) {
+          var newImageSrc = this.newSrc(imageSrc, imgWidth, imgHeight, $image[0]);
```

Comment:
That's a no-op essentially.

Reply:
I don't really remember the thought process, but I probably didn't know what newSrc does so added this...